### PR TITLE
Radios & Flares

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Adds CM flares and radiophones to their respective pouches :)

## Why / Balance
#560 & #561

## Technical details
YAML and Sprites

## Media
image
image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
🆑
add: Marine Radiophones, a military grade walkie talkie.
add: Marker Flares, light up the night.

### CODE REVIEW

**Comment by KittenColony** [comment]
why the fuck are flares so difficult :(

**Comment by KittenColony** [comment]
why the fuck am I doing this, we already have all the pouches asked for in #520, do I just need to give them fills?

**Comment by KittenColony** [comment]
flare might be too bright :clueless:

**Comment by KittenColony** [comment]
heads up, flares are very red and I don't know what you'd want them to be changed to

**Comment by QuietlyWhisper** [comment]
these sprites should be vertical otherwise they don't fit properly in the ui

**Comment by QuietlyWhisper** [comment]
image

**Comment by QuietlyWhisper** [comment]
I can probably fix it though, but I find it stupid that SS14 even needs "storage" sprites, seems like a waste of space

**Comment by QuietlyWhisper** [comment]
storage
You'll need this

File: Resources/Prototypes/_CM14/Entities/Objects/Tools/flares.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      - map: [ enum.ExpendableLightVisualLayers.Overlay ]
+        state: flare_unlit
+        color: "#FFcccc"
+  - type: Icon
+    sprite: _CM14/Objects/Misc/flares.rsi
+    state: icon
+  - type: Item
+    sprite: _CM14/Objects/Misc/flares.rsi
+    heldPrefix: unlit
+    size: Tiny
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Objects/Tools/flares.yml:42]:
```suggestion
    size: Small
    storedSprite:
      sprite: _CM14/Objects/Misc/flares.rsi
      state: storage
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_CM14/Entities/Objects/Tools/flares.yml (hunk 1 above)
Comment (QuietlyWhisper) [reply Resources/Prototypes/_CM14/Entities/Objects/Tools/flares.yml:42]: You also need to update the meta.json obviously

**Comment by QuietlyWhisper** [comment]
Other than that everything looks fine thanks!

**Comment by DrSmugleaf** [comment]
> I find it stupid that SS14 even needs "storage" sprites, seems like a waste of space

You don't need one if it is the same as the regular sprite. If it's not specified it defaults to the one in the sprite component. The only case to use one is if you want the item to use a different sprite when stored, such as when the sprite doesn't fit the boundaries in the storage.

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**